README.md

```diff
@@ -911,20 +911,3 @@ Protocol tests can be found in the `sdk/test/ProtocolTests` directory. Protocol
 
 All low-level service clients are created using the code generator found in the **generator** folder. The code generator 
 uses the service models defined in the **generator\ServiceModels** folder.
-
-[nuget-info]: https://nuget.org/
-[aws]: http://aws.amazon.com/
-[sdk-website]: http://aws.amazon.com/sdkfornet
-[sdk-forum]: http://developer.amazonwebservices.com/connect/forum.jspa?forumID=61
-[sdk-source]: https://github.com/aws/aws-sdk-net
-[sdk-issues]: https://github.com/aws/aws-sdk-net/issues
-[sdk-license]: http://aws.amazon.com/apache2.0/
-[docs-api]: http://docs.aws.amazon.com/sdkfornet/v3/apidocs/Index.html
```

Comment:
I found this by searching for `Index.html`. This is invalid markdown, it doesn't get rendered today so I removed all these links.